storage/redistore/peer_store.go

```diff
@@ -0,0 +1,169 @@
+//Note: ip6 seperation into shards is unnecessary when using Redis(?)
+package redistore
+
+import (
+	"encoding/binary"
+	"log"
+	"time"
+
+	"github.com/chihaya/chihaya/bittorrent"
+	"github.com/chihaya/chihaya/storage"
+	"github.com/garyburd/redigo/redis"
+)
+
+var namespacePrefix string
+
+type Config struct {
+	Namespace                 bool          `yaml:"namespace"`
+	Cntrl                     string        `yaml:"prefix"`
```

Comment:
Because it could be a prefix or a database instance(0-16) based on the Namespace yes or no bool. I should think of a better name.